tasks/gulp/__tests__/after-build-package.test.js

```diff
@@ -1,112 +1,83 @@
 const { readFile } = require('fs/promises')
-const path = require('path')
-const recursive = require('recursive-readdir')
-const glob = require('glob')
+const { join, normalize } = require('path')
+const slash = require('slash')
 
 const configPaths = require('../../../config/paths.js')
-const { getDirectories, getFilesByDirectory } = require('../../../lib/file-helper')
+const { filterPath, getDirectories, getListing, mapPathTo } = require('../../../lib/file-helper')
 const { componentNameToJavaScriptClassName, componentNameToJavaScriptModuleName } = require('../../../lib/helper-functions')
 
 const { renderSass } = require('../../../lib/jest-helpers')
 
 describe('package/', () => {
+  let listingSource
+  let listingPackage
+
   let componentsFilesSource
   let componentsFilesPackage
   let componentsFilesPackageESM
 
   let componentNames
 
   beforeAll(async () => {
-    componentsFilesSource = await getFilesByDirectory(configPaths.components)
-    componentsFilesPackage = await getFilesByDirectory(`${configPaths.package}govuk/components/`)
-    componentsFilesPackageESM = await getFilesByDirectory(`${configPaths.package}govuk-esm/components/`)
+    listingSource = await getListing(join(configPaths.src, '../'))
```

Comment:
We may have a slight naming issue there, as I thought `configPaths.src` was the `src` folder, while it's actually `src/govuk`. 

*suggestion*: I think a little comment will help the next person reading that, (or maybe `join(cwd(), 'src')` to get the path), as a quick way out.

```suggestion
    // Join with `../` to get the content of the `src` directory
    listingSource = await getListing(join(configPaths.src, '../'))
```